subnet/subnet.go

```diff
@@ -32,8 +32,10 @@ type LeaseAttrs struct {
 
 type Lease struct {
 	Subnet     ip.IP4Net
-	Attrs      *LeaseAttrs
+	Attrs      LeaseAttrs
 	Expiration time.Time
+
+	asof uint64
```

Comment:
You're right. I started with a branch for doing reservations and then broke it apart into what is now 4 separate PRs (this is 2nd). The `asof` is only needed in the last one, the one doing reservations. I guess I could yank it here and add it into the last one where it's needed but it's not doing harm so can we let it hang out here for just a bit?